lib/ossert/fetch/stackoverflow.rb

```diff
@@ -0,0 +1,43 @@
+module Ossert
+  module Fetch
+    class StackOverflow
+      attr_reader :client, :project
+
+      extend Forwardable
+      def_delegators :project, :agility, :community, :meta
+
+      BASE_URL = 'https://api.stackexchange.com/2.2/'
+      SEARCH_URL = 'search/advanced'
+      SEARCH_PARAMS = {
+        site: :stackoverflow,
+        tagged: 'ruby',
+        filter: :total,
+        key: ENV['SO_TOKEN']  # optional, but without it only 300 requests/day are available
```

Comment:
There is a problem I think. 
We send parameter ```key``` in any scenario, even when it has blank value.
I think it is a bit frustrating.